api/services/ocean/spark/schemas/oceanSparkClusterConfigLogCollection.yaml

```diff
@@ -6,6 +6,13 @@ properties:
     type: boolean
     default: true
     example: false
+    deprecated: true
```

Comment:
we should probably change the description as well to tell users to use `collectAppLogs` instead